editor.planx.uk/src/@planx/components/FindProperty/model.ts

```diff
@@ -1,5 +1,12 @@
 import { MoreInformation, parseMoreInformation } from "../shared";
 
+export enum FindPropertyUserAction {
+  Existing = "Selected an existing address",
+  New = "Proposed a new address",
+}
+
+export const PASSPORT_COMPONENT_ACTION_KEY = "findProperty.action";
```

Comment:
Creating a new passport variable, rather than using `_address.source` has two main benefits to keep this quick & simple:
- The values can be human readable in the source, rather than just "os" & "proposed"
- We don't have to refactor allow list handling to work with deconstructed, nested passport variables like `_address`